### ISSUE TITLE
Relaybot not working with WA Business

### ISSUE THREAD

**Issue by topikuning** [mautrix/whatsapp#225]
I have spent 2 days to implement the relaybot, eventually, the problem is in WA account type.
I change from business to normal one, and everything is work normally.

The problem when using WA Business in relaybot mode is, we can not send matrix message to Whatsapp, but whatsapp message can send to matrix.

EDIT:
it is look like working, but the strange is in every new message from new contact, the matrix user message not send to whatsapp, but when in the management room, send !wa reconnect command, the message then send.
but I have to do it manually, every single new contact and message

**Comment by chagai95** [comment]
experienced the exact same issue, thanks so much for opening this issue!

adding the log message so people can find when they google:

Ignoring event $6K2CeOcGzTtkVbgtbf1poZZUpqMIgYffdvjdsfnkvcu3WBEe4 from @username:domain as user has no session

**Comment by chagai95** [comment]
the same thing happened to me on regular wa so I'm not sure it's the same issue anymore...

**Comment by dbedoya** [comment]
we have the same issue with regular WhatsApp using relaybot enabled, and we've been able to make it work by inserting a record in the table `user_portal` just after the room for a new chat is created. Without that record the messages sent to a new user that writes from their WhastApp are not bridged. We found out that the reason of that is the function `portal.HasRelaybot()` returning `false` when it is called inside `HandleMatrixMessage` (in portal.go), preventing the message from being handle by the bridge.
If any of you has any other ideas to fix this issue it would be great to hear from you.
